scene/animation/tween.cpp

```diff
@@ -312,6 +316,12 @@ bool Tween::step(float p_delta) {
 				start_tweeners();
 			}
 		}
+		
+#ifdef DEBUG_ENABLED
+		if (Math::is_equal_approx(rem_delta, prev_delta) && running && loops <= 0) {
+			ERR_FAIL_V_MSG(true, "Infinite loop detected. Check set_loops() description for more info.");
```

Comment:
Wouldn't it be worth returning `false` here to make it abort?
Spamming with errors can make very big log files and fill users' disks.

Reply:
That's what I did at first, but it finished the tweening abruptly and a single error is hard to miss; it thought it could be confusing. But yeah, considering the logs, it's better not to spam.